app/models/rook.rb

```diff
@@ -1,3 +1,26 @@
 class Rook < Piece
 
+	def valid_move?(dest_x, dest_y)
+
+		#a move can not be a non-move
+    	return false if dest_x == self.x_position && dest_y == self.y_position
+
+		#no obstructions allowed
+		return false if is_obstructed?(dest_x, dest_y) == true
```

Comment:
@robgraphix1 Whoops! Forget what I just said. Sorry, not all my brain's cylinders are firing.